daemon/mgr/spec_nvidia_hook.go

```diff
@@ -0,0 +1,40 @@
+package mgr
+
+import (
+	"os/exec"
+
+	"github.com/alibaba/pouch/pkg/utils"
+
+	"github.com/opencontainers/runtime-spec/specs-go"
+)
+
+var (
+	nvidiaHookName = "nvidia-container-runtime-hook"
```

Comment:
please add more description to this variable. Is this is local file when executing hook?

Reply:
Both nvidia-container-runtime-hook and nvidia-container-cli was packaged in pouch rpm.